curriculum/challenges/english/16-the-odin-project/top-learn-block-and-inline/learn-block-and-inline-question-e.md

```diff
@@ -0,0 +1,31 @@
+---
+id: 6542214750bc5c6ed8abf74a
+title: Learn Block and Inline Question E
+challengeType: 15
+dashedName: learn-block-and-inline-question-e
+---
+# --description--
+
+You can't talk about `block` and `inline` elements without discussing `div`s and `span`s. All the other HTML elements you have encountered so far give meaning to their content. For example, paragraph elements tell the browser to display the text it contains as a paragraph. Strong elements tell the browser which texts within are important and so on. Yet, divs and spans give no particular meaning to their content. They are just generic boxes that can contain anything.
+
+# --question--    
+
+## --text--
+
+What is the primary difference between `span` and `div` elements from other elements?
```

Comment:
```suggestion
What is the primary difference of `span` and `div` elements from other elements?
```

"Difference between" implies we are comparing the `span` with the `div`.

Reply:
I cheated and asked my wife: "What is the main way in which `span` and `div` elements differ from other elements?".

Reply:
I like that suggestion.